### PR TITLE
Simplify telegram sink

### PR DESCRIPTION
**Pull request by arikalon1**

### CODE REVIEW

**Comment by aantn** [comment]
The doc changes look good. Haven't had a chance to look over the other parts yet.

File: src/robusta/core/sinks/telegram/telegram_client.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            file_name = file_name.replace(SVG_SUFFIX, PNG_SUFFIX)
+
+        files = {
+            file_type.lower(): (file_name, contents)
+        }
+        response = requests.post(url, files=files)
+
+        if response.status_code != 200:
+            logging.error(
+                f"Failed to send telegram file: chat_id - {self.chat_id} reason - {response.reason} {response.text}"
```
Comment (kotlickya) [inline src/robusta/core/sinks/telegram/telegram_client.py:44]: i would add response.status code code too here

Comment (kotlickya) [reply src/robusta/core/sinks/telegram/telegram_client.py:44]: also above

Comment (arikalon1) [reply src/robusta/core/sinks/telegram/telegram_client.py:44]: we have it in the response text

**Pull request merged**